README.md

```diff
@@ -22,6 +22,29 @@ choco install kyma-cli
 
 Read more about [installation options](https://github.com/kyma-project/cli/blob/master/docs/04-01-installation.md).
 
+### Build from Sources
+
+Alternatively, you can also build the Kyma CLI from the sources:
+
+1. To clone the Kyma CLI repository, run:
+
+```bash
+git clone git@github.com:kyma-project/cli.git
```

Comment:
```suggestion
mkdir -p $GOPATH/src/github.com/kyma-project/
git clone git@github.com:kyma-project/cli.git $GOPATH/src/github.com/kyma-project/cli
```

Reply:
Would be good to point out that the sources have to fit into the Golang folder structure.